bridges/scalajs-1/src/main/scala/bloop/scalajs/JsBridge.scala

```diff
@@ -51,6 +53,46 @@ object JsBridge {
       }
     override def trace(t: => Throwable): Unit = logger.trace(t)
   }
+  private case class LinkerInput(
+      isFullLinkJS: Boolean,
+      moduleKind: ModuleKindJS,
+      target: Path
+  )
+  private object ScalaJSLinker {
+    private val cache = TrieMap.empty[LinkerInput, WeakReference[Linker]]
+    def reuseOrCreate(input: LinkerInput): Linker =
+      if (input.isFullLinkJS) createLinker(input)
+      else {
+        cache.get(input) match {
+          case Some(WeakReference(linker)) => linker
+          case _ =>
+            val newLinker = createLinker(input)
+            cache.update(input, WeakReference(newLinker))
+            newLinker
+        }
+      }
+    private def createLinker(input: LinkerInput): Linker = {
+      val semantics = input.isFullLinkJS match {
+        case true => Semantics.Defaults.optimized
+        case false => Semantics.Defaults
+      }
+      val scalaJSModuleKind = input.moduleKind match {
+        case ModuleKindJS.NoModule => ScalaJSModuleKind.NoModule
+        case ModuleKindJS.CommonJSModule => ScalaJSModuleKind.CommonJSModule
+        case ModuleKindJS.ESModule => ScalaJSModuleKind.ESModule
+      }
+
+      val useClosure = input.isFullLinkJS && input.moduleKind != ModuleKindJS.ESModule
+
+      val config = StandardConfig()
+        .withOptimizer(true)
```

Comment:
`.withOptimizer(true)` should be used also with `fastLinkJS`.

Reply:
`withOptimizer(true)` is the default anyway. ;)